CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+Contributions, issues and feature requests are very welcome. If you are using this app and fixed a bug or improve it, please consider submitting a PR!
+
+## Guidelines
+
+### Commit messages
+
+This project uses [gitmoji](https://gitmoji.carloscuesta.me/) as commit convention, because it's fun and provides an easy way of identifying the purpose or intention of a commit with only looking at the emojis used. All you have to do is to find the emoji corresponding to the purpose of your commit in [this list](https://gitmoji.carloscuesta.me/) following by a short message explaining the update. You can also add a description if it's needed. That's all ;)
```

Comment:
```suggestion
This project uses [gitmoji](https://gitmoji.carloscuesta.me/) as the commit convention, because it's fun and provides an easy way of identifying the purpose or intention of a commit only by looking at the emojis used. All you have to do is to find the emoji corresponding to the purpose of your commit in [this list](https://gitmoji.carloscuesta.me/), followed by a short message explaining the update. You can also add a description if it's needed. That's all ;)
```

Just grammar fixes :)